scripts/ci/perf-tests/iot.yaml

```diff
@@ -0,0 +1,94 @@
+---
+name: iot 
+start_time: "2018-01-01T00:00:00Z"
+end_time: "2018-01-01T12:00:00Z"
+data:
+  type: bulk_data_file_loader
+query_tests:
+  # influxql versions (http means influxql)
+  - {"type": "build_query_file", "format": "http", "use_case": "window-agg", "query_type": "min"}
```

Comment:
I was tempted to write a separate `query_matrix_test` for this, but decided to keep it simple - there's only 80 lines here.